src/core/Notification/Notification.test.tsx

```diff
@@ -0,0 +1,141 @@
+import { Notification } from './Notification';
+import { axeTest } from '../../utils/test';
+
+describe('Error view tests', () => {
+  const ErrorNotification = (
+    <Notification
+      status="error"
+      className="notification-test-class"
+      closeText="Close"
+      smallScreen
+      labelText="Lorem ipsum dolor sit"
+    >
+      Testcontent
+    </Notification>
+  );
+
+  it('should have status and smallScreen prop specific classNames', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('fi-notification--error');
+    expect(container.firstChild).toHaveClass('fi-notification--small-screen');
+  });
+  it('should have given className as props', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('notification-test-class');
+  });
+});
+describe('props', () => {
+  describe('children', () => {
+    const notificationWithElementChild = (
+      <Notification closeText="Close" labelText="Lorem ipsum dolor sit">
+        <div>Test notification</div>
+      </Notification>
+    );
+
+    it('has the given content', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container.firstChild).toHaveTextContent('Test notification');
+    });
+
+    it('should match snapshot', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('classnames', () => {
+    const customClassNotification = (
+      <Notification
+        closeText="Close"
+        className="custom-class"
+        labelText="Lorem ipsum dolor sit"
+      >
+        Test content
+      </Notification>
+    );
+
+    it('contains base classname', () => {
+      const { container } = render(customClassNotification);
+      expect(container.firstChild).toHaveClass('fi-notification');
+    });
+
+    it('contains custom classname', () => {
+      const { container } = render(customClassNotification);
+      expect(container.firstChild).toHaveClass('custom-class');
+    });
+  });
+  const NotificationWithButtonProps = (
+    <Notification
+      labelText="Lorem ipsum dolor sit"
+      closeText="Close"
+      closeButtonProps={{
+        'aria-labelledby': 'test-element',
+        disabled: true,
+        className: 'testClass',
+      }}
+    >
+      Testcontent
+    </Notification>
+  );
+  it('Button should contains custom classname', () => {
+    const { getByRole } = render(NotificationWithButtonProps);
+    expect(getByRole('button')).toHaveClass('testClass');
+  });
+  it('Button should have given props', () => {
+    const { getByRole } = render(NotificationWithButtonProps);
+    expect(getByRole('button')).toHaveAttribute(
+      'aria-labelledby',
+      'test-element',
+    );
+    expect(getByRole('button')).toHaveAttribute('disabled');
+    expect(getByRole('button')).toHaveTextContent('Close');
+  });
```

Comment:
As we are testing the `closeButtonProps` prop here it would be wise to have these test inside it's own `describe` block like e.g `className` prop above.

Reply:
Fixed. Is now in own describe.